scaleway/helpers_lb.go

```diff
@@ -67,3 +67,69 @@ func flattenLbStickySessionsType(t lb.StickySessionsType) interface{} {
 func expandLbStickySessionsType(raw interface{}) lb.StickySessionsType {
 	return lb.StickySessionsType(raw.(string))
 }
+
+func flattenLbHCTCP(config *lb.HealthCheckTCPConfig) interface{} {
+	if config == nil {
+		return nil
+	}
+	return []map[string]interface{}{
+		{},
```

Comment:
why is it empty ?

Reply:
Because `HealthCheckTCPConfig` is an empty struct